index.html

```diff
@@ -254,9 +261,9 @@ <h3>Audio stream with titles and images</h3>
       <section>
         <h3>Control messages for media streaming clients</h3>
         <p>
-          A media streaming server uses <a>media timed events</a> to send control
+          A media streaming server uses <a>timed metadata</a> to send control
```

Comment:
Since these messages are being used to control behaviour, calling them "metadata" is a stretch.

Reply:
I agree, so I have reverted to the previous text.